conda-forge.yml

```diff
@@ -3,5 +3,6 @@ appveyor:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
+provider: {linux_aarch64: azure, linux_ppc64le: azure}
```

Comment:
```suggestion
provider: {linux_aarch64: azure, linux_ppc64le: default}
```
ppc64le builds fail on azure. They need to run on travis (`default`, or `travis`)

Reply:
actually they're not failing anymore, so we might not need to